spec/support/ems_folder_helper.rb

```diff
@@ -0,0 +1,6 @@
+def vmware_nested_folders(ems)
```

Comment:
@bdunne I don't think we do that elsewhere.  While I don't disagree with you, I think this would be a new thing.  The only other thing we have is the EvmSpecHelper (which this could reopen or nest a module inside of)